### PR TITLE
Power Computer and Computer Fixes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Marked as high priority because map-fixes shouldn't hold up future map editing.

Fixin' timeee

- Fixes: https://github.com/ParadiseSS13/Paradise/issues/1508 missing atmos tank stand
- Fixes: https://github.com/ParadiseSS13/Paradise/issues/1420 power monitor lack of APC area
 - Should be a lot easier to find errors of duplicate APCs in an area (like the two instances of surgery in this example).
- Fixes: Two APCs that would never charge
- Converts power monitors over to *proper* computers, complete with utilizing keyboard and screen overlays.

yay!

### CODE REVIEW

**Comment by Taaketa** [comment]
Yes yes yes!!!

File: code/game/machinery/computer/power.dm [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
-	icon_state = "power"
-	density = 1
-	anchored = 1
+	icon_screen = "power"
+	icon_keyboard = "power_key"
 	use_power = 2
 	idle_power_usage = 20
 	active_power_usage = 80
-
-	light_color = LIGHT_COLOR_ORANGE
```
Comment (tigercat2000) [inline code/game/machinery/computer/power.dm]: why was this removed

**Comment by KasparoVy** [comment]
Oh thank you lord the computer will finally work

<3
